active/ThemingCornerRadii/CornerRadiusAPI.md

```diff
@@ -0,0 +1,175 @@
+
+|#| Control | Lightweight resource(s) | Affected template attribute/style |
+|:-:|:-|:-|:-|
+|1| Button | ButtonThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|2| Checkbox | CheckboxThemeCornerRadius | RootGrid (Grid), NormalRectangle (Checkbox Box Rectangle) |
+|3| TextBox | TextBoxThemeCornerRadius | BorderElement (Border) |
+|4| AutoSuggestBox | AutoSuggestBoxThemeCornerRadius | AutoSuggestBoxTextBoxStyle (CornerRadius Property definition) |
+|5| Slider | SliderThemeCornerRadius | HorizontalTrackRect, HorizontalDecreaseRect, VerticalTrackRect, VerticalDecreaseRect (All Rectangles) |
+|6| ComboBox | ComboBoxThemeCornerRadius, ComboBoxItemThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|7| DatePicker | DatePickerThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|8| TimePicker | TimePickerThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|9| CalendarDatePicker | CalendarDatePickerThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|10| ProgressBar | ProgressBarThemeCornerRadius | DeterminateRoot (Border) ProgressBarIndicator (Rectangle) |
+|11| RichEditBox | RichEditBoxThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|12| ToggleButton | ToggleButtonThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|13| ToggleSwitch | ToggleSwitchThemeCornerRadius | OuterBorder (Rectangle), SwitchKnobBounds (Rectangle) |
+|14| SplitButton | SplitButtonThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|15| DropDownButton | DropDownButtonThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|16| PasswordBox | PasswordBoxThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|17| RichEditBox | RichEditBoxThemeCornerRadius | CornerRadius Property (Setter in Style) |
+|18| Tab Control | TabControlThemeCornerRadius | *tbd* |
+|19| ToggleSplitButton | ToggleSplitButtonThemeCornerRadius | CornerRadius Property (Setter in Style) *Top,Bottom only on one side for each button "split"* |
+|20| FlipView | FlipViewThemeCornerRadius | *tbd* |
+|21| GridView | GridViewThemeCornerRadius | *tbd* |
+|22| ListView | ListViewThemeCornerRadius | *tbd* |
+|23| TreeView | TreeViewThemeCornerRadius | *tbd* |
+
+#### Control "Bars"
+The following controls receiving rounded corners don't back backplates that can round per-say, but instead have visual components (dubbed "bars") that indicate selection or state that will be getting rounded.
+
+|#| Control | Lightweight resource(s) | Affected template attribute/style |
+|:-:|:-|:-|:-|
+|1| NavigationView | NavigationViewSelectorThemeCornerRadius | NavigationViewItemPresenter *(Top and Left style)* SelectionIndicator (Rectangle) |
+|2| Pivot | PivotSelectorThemeCornerRadius | FocusFollower (Rectangle) |
+|3| ScrollIndicator | ScrollIndicatorThemeCornerRadius | *tbd* |
+|4| ColorPicker | ColorPickerThemeCornerRadius | *tbd* |

```

Comment:
This one is a composite of all sorts of controls and its own thing. I've been trying to track the fine grain list of changes in the figma over in [this WinUI issue](https://github.com/microsoft/microsoft-ui-xaml/issues/659).

Reply:
Sorry I can't see the direct control you've highlighted - which control is a composite, or were you referring to all the controls under "bars".

Reply:
I believe @chrisglein is talking about ColorPicker.